src/components/bcros/Alert.vue

```diff
@@ -0,0 +1,103 @@
+<template>
```

Comment:
I think its worth putting this inside 'bcros/alert/Index.vue' and then having the 'Alerts.vue' -> 'bcros/alert/List.vue' since they are sort of grouped together. What do you think?

Reply:
Agree with this, I think we are doing most of the stuff like that.
Also, I would maybe call it 
BusinessAlerts or DashboardAlerts or smth like that rather then just alerts, it could clash the naming with other types of alerts.
@kialj876 what do you think about naming it bit different ?

Reply:
This component is more generic than that, it can also take severity, text and description and then it works as a regular alert, the business specific stuff is in dashboard and alertType: frozen for instance

Reply:
Yeah you're right and we can add a slot over the contact info if we want it to handle something else later. I am good with it staying as BcrosAlert

Reply:
OK, like we discussed in chat, we can update naming in the future.
Also, frozen sound like business status, so I would try to keep that out of alert itself, and let parent component handle how to display it.

Reply:
Either way, I am fine for this PR as is, so you can resolve this comment.

Reply:
yeah it looks like the other 'alert types' in the enum are business specific too, but we can update the naming later